hs_discover/src/components/Search.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div id="discover-search">
+        <resource-listing :key="resources"
+                          :columns="gridColumns"
+                          :labels="gridColumnLabels">
+        </resource-listing>
+    </div>
+</template>
+
+<script>
+import Resources from './Resources.vue';
+
+export default {
```

Comment:
should populating of the geodata array happen here instead of Resource.vue?  This doesn't feel like a complete component.

Reply:
This is vestigial code from a past version of Discover rewrite that more closely matched what we have in production. Just cleaned up now and you'll see in the commit for addressing PR change requests.

Reply:
At this point I've rewritten areas of Discover so many ways, the cleanup this time took a bit longer and this apparently was missed.

Reply:
See commit 6ce5346f10a3a862918aeb20679ff173e26d9f0b

Reply:
need to push?